cloudpassage/event.py

```diff
@@ -48,6 +49,8 @@ def list_all(self, pages, **kwargs):
             server_id (list or str): A list or comma-separated string \
             containing the server IDs to retrieve events for.
             server_platform (str): (linux | windows)
+            critical (true | false): Returns only critical or \
```

Comment:
Should be (bool) instead of (true | false).  See https://pythonhosted.org/an_example_pypi_project/sphinx.html#full-code-example

Otherwise, looks good.  Please correct this and I'll accept.